addons/sale/static/src/js/sale_order_line_field/sale_order_line_field.js

```diff
@@ -46,18 +46,33 @@ export class SaleOrderLineListRenderer extends ProductLabelSectionAndNoteListRen
      */
 
     /**
-     * Whether the provided record is a section, a note, or a combo.
-     *
-     * This method's name isn't ideal since it doesn't mention combos, but we'd have to override a
-     * few other methods to fix this, and the added complexity isn't worth it.
-     *
-     * @param record The record to check
-     * @return {Boolean} Whether the record is a section, a note, or a combo.
-     */
+    * Whether the provided record is a section, a note, or a combo.
+    *
+    * This method's name isn't ideal since it doesn't mention combos, but we'd have to override a
+    * few other methods to fix this, and the added complexity isn't worth it.
+    *
+    * @param record The record to check
+    * @return {Boolean} Whether the record is a section, a note, or a combo.
+    */
     isSectionOrNote(record=null) {
         return super.isSectionOrNote(record) || this.isCombo(record);
     }
 
+    getSectionColumns(columns) {
+        return columns;
+    }
+
+    getCellClass(column, record) {
+        let classNames = super.getCellClass(column, record);
+        const sectionColumns = ['product_uom_qty', 'discount']
+        if (this.isSectionOrNote(record) && sectionColumns.includes(column.name)) {
+            return classNames.replace(" o_hidden", "");
+        }else if (this.isSectionOrNote(record) && !sectionColumns.includes(column.name)) {
+            return classNames.replace("o_hidden", "invisible");
+        }
+        return classNames;
+    }
```

Comment:
Seems wrong :thinking: 

Because of this qty and discount are displaying in section also
![image](https://github.com/user-attachments/assets/8c83a989-6e66-4b1c-b368-309917853ac4)